.gitignore

```diff
@@ -0,0 +1,8 @@
+#enviroment
+.env
+
+#pycache
+.pytest_cache/
```

Comment:
Just use `__pycache__/` is good enough.
Refer to:  https://github.com/tonytan4ever/py-logwatcher/blob/master/.gitignore
You can directly copy && paste here..